pinterest_test.go

```diff
@@ -1384,25 +1342,13 @@ func (suite *ClientTestSuite) TestUnauthorizedMeSearchBoardsFetch() {
 // the pins of the authorized user
 func (suite *ClientTestSuite) TestSuccessfulMeSearchPinsFetch() {
 	// Load first page
-	pins, page, err := suite.client.Me.Search.Pins.Fetch(
+	_, _, err := suite.client.Me.Search.Pins.Fetch(
 		"Go Gopher",
 		&controllers.MeSearchPinsFetchOptionals{
 			Limit: 1,
 		},
 	)
```

Comment:
Search is now only searching against pins that you have created, not only re-pinned.

I will need to create a pin that I always have to unit test this.